mysqld_exporter.go

```diff
@@ -178,6 +178,16 @@ func main() {
 		}
 	}
 	http.HandleFunc(*metricPath, prometheus.InstrumentHandlerFunc("metrics", newHandler(enabledScrapers)))
+        // add an endpoint for mysql health state
+	http.HandleFunc("/mysqlhealth", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
/-/healthy is our standard health endpoint: https://github.com/prometheus/prometheus/blob/master/web/web.go#L308